src/components/nav-menu/index.js

```diff
@@ -62,17 +62,34 @@ const NavMenu = ({summit, active, user, exclusiveSections, globalSummitDocs}) =>
                             active={activeItem === it.name}
                         />
                     ))}
-                    {globalSummitDocs?.map(doc => (
-                        <MenuItem
-                            key={doc.name}
-                            name={doc.name}
-                            label={doc.label}
-                            iconClass="fa-download"
-                            show
-                            onClick={(e) => window.open(doc.file, '_blank')}
-                            active={false}
-                        />
-                    ))}
+                    {globalSummitDocs?.map(doc => {
+                        const res = []
+                        if (doc.file) {
```

Comment:
refactor this to avoid duplication u could have a single if based on doc.file or doc.weblik and map an object with the properties populatd for one case or another
the render of the menu item is the same

Reply:
done